firebase-firestore/src/main/java/com/google/firebase/firestore/core/Query.java

```diff
@@ -507,16 +507,6 @@ public Target toTarget() {
     return this.memoizedTarget;
   }
 
-  /** Returns true if the query contains any composite filters (AND/OR). Returns false otherwise. */
-  public boolean containsCompositeFilters() {
```

Comment:
Was this not used?

Reply:
There was a PR that removed the one usage of this function, but that PR failed to recognize that that was the only usage and forgot to remove this function along with it.